git/mockgit/mockgit.go

```diff
@@ -42,6 +42,11 @@ func (m *Mock) Git(args string, output *string) error {
 	return nil
 }
 
+func (m *Mock) ExpectationsMet() {
```

Comment:
This ensures that all expectations have been met. It makes it possible to avoid extra ExpectX calls.